symengine/symengine_exception.h

```diff
@@ -0,0 +1,16 @@
+#ifndef SYMENGINE_EXCEPTION_H
+#define SYMENGINE_EXCEPTION_H
+
+namespace SymEngine
+{
+
+class DivisionByZero: public std::exception
+{
+    virtual const char* what() const throw()
+    {
+    return "Division By Zero";
+    }
```

Comment:
I am confused. Shouldn't the function be defined only once? And there are two namespaces `symengine` and `SymEngine`.